exchanges/binance.js

```diff
@@ -134,14 +134,14 @@ Trader.prototype.getPortfolio = function(callback) {
 
     if (!_.isNumber(assetAmount) || _.isNaN(assetAmount)) {
       log.error(
-        `Binance did not return portfolio for ${this.asset}, assuming 0.`
+        'Binance did not return portfolio for ${this.asset}, assuming 0.'
```

Comment:
Won't print out correctly, you need backticks for the inlined variables.

Reply:
well that was silly of me